animation.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+
+      .container {
+        width: 100%;
+        height: 80vh;
+        background-image: url("d-wallpaper-bg.jpg");
+        position: relative;
+        overflow-x: hidden;
+      }
+
+      .fog {
+        height: 300px;
+        width: 500%;
+        display: block;
+        background-image: url("fog.png");
+        top: 0;
+        left: 0;
+        right: 0;
+        background-repeat: repeat-x;
+        animation: fog 8s linear infinite;
+      }
+
+      @keyframes fog {
+        100% {
+          transform: translateX(-300px);
+        }
+      }
+
+      .blue-car {
+        left: 75%;
+        position: absolute;
+        bottom: 210px;
+      }
+
+      .blue-car img {
+        width: 100%;
+        height: 150px;
+        animation: bluecar 5s linear infinite;
+        -webkit-animation-iteration-count: 1;
```

Comment:
can be used in shorthand directly if using webkit add support for all properties